docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/conversions/index.md

```diff
@@ -166,7 +166,108 @@ Using our [Snowplow e-commerce tracking](/docs/collecting-data/collecting-from-o
 
 ### Configuration Generator
 
-You can use the full config generator on the [configuration page](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/unified/index.mdx) to help you generate your conversion event definitions.
+You can use the below generator to generate the conversion events variable, make sure you combine this with existing conversions if you have any already. You can also use the full config generator on the [configuration page](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/unified/index.mdx) to help you generate all your package variables.
+
+```mdx-code-block
+import CodeBlock from '@theme/CodeBlock';
+import { useState } from 'react';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import { useColorMode } from '@docusaurus/theme-common';
+import validator from '@rjsf/validator-ajv8';
+import Form from '@rjsf/mui';
+
+export const schema = {
+  "recommendFullRefresh": false,
+      "title": "Conversion Definition",
+      "description": "> Click the plus sign to add a new entry",
+      "type": "array",
+      "minItems": 0,
+      "items": {
+        "type": "object",
+        "required": [
+          "name",
+          "condition"
+        ],
+        "title": "",
+        "description": "Conversion Event",
+        "properties": {
+          "name": {
+            "type": "string",
+            "title": "Name",
+            "description": "Name of your conversion type"
+          },
+          "condition": {
+            "type": "string",
+            "title": "Condition",
+            "description": "SQL condition e.g. event_name = 'page_view'"
```

Comment:
Nit: The descriptions are a bit confusing to me – they look like headings and it's not clear if they belong to the input above or below. To me it would be more clear if they were above the input or if the description was part of the title.

Reply:
I get your point, but I'm going to leave it so it's aligned with the other form. To be honest the formatting for this whole widget kind of sucks and varies depending on the type of the field (boolean description is in completely different place to numeric for example), so if you're ever bored you're welcome to mess with it all!